.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,9 +19,9 @@ import holodeck
 A clear and concise description of what you expected to happen.
 
 **Logs**
-```
 Please include the client side output and the Engine debug log.
-```
+ - Engine Debug Log (in the path of `holodeck.util.logs_path()`)
+
```

Comment:
Seems like there's redundant information here. Maybe you could consolidate everything on the same line?